docs/user_guide.rst

```diff
@@ -4,6 +4,25 @@ User Guide
 General Information
 *********************
 
+.. dropdown:: Tips for Input
+
+    .. role:: python(code)
+        :language: python
+    
+    Sparcle_QC depends on a properly formatted PDB file. Make sure your PDB follows the standard format. Sparcle_QC does not require occupancy, temperature factor, segment identifier, or formal charge, but it does require all other columns. 
+
+    **CHARMM vs Amber**
+
+    If using CHARMM, add a psf file (downloaded from CHARMM-GUI) to your working directory. Point to the psf file in the Sparcle_QC input with keyword :python:`charmm_psf`. Add the protein PDB's path under the :python:`pdb_file` keyword. Separately add the ligand named "ligand.pdb" to your working directory.
+
+    If using Amber, the :python:`pdb_file` path must point to a PDB with the complex (both the protein and ligand included). Make sure the ligand atoms have "HETATM" in columns 1-6 of the PDB.
```

Comment:
can we test super quick if it has to be HETATM. i know in reorder atoms i make sure that it is HETATM and i think that autocap will be able to pull out the ligand regardless because it hinges upon 'NOT' logic: like checking that it is not protein and not solvent and not metals, ...